android/src/google/play/listings/en-US/full-description.txt

```diff
@@ -42,7 +46,7 @@ At Organic Maps, we believe that <b>privacy is a fundamental human right</b>:
 
 Zero trackers and only minimally required permissions are found according to Exodus Privacy Report.
 
-Please visit <b><i>organicmaps.app</i></b> website for additional details.
+Please visit <b><i>organicmaps.app</i></b> website for additional details and FAQ, and contact us directly at @OrganicMapsApp in Telegram.
```

Comment:
Nobody cares about Telegram outside Russian-speaking countries.

Reply:
The main argument is that it's our active support channel, where our users can help us to support others.

Reply:
ok, np